docs/core/tools/dotnet-workload-sets.md

```diff
@@ -0,0 +1,167 @@
+
+```dotnetcli
+dotnet workload update --version 8.0.400
+```
+
+Workload-set update mode will be selected if it wasn't already selected.
+
+## Use global.json for the workload set version
+
+Create a `global.json` file to specify the workload set version for a repository. The format looks like this example:
+
+```json
+{
+  "sdk": {
+    "workloadVersion": "9.0.100-preview.7.24414.1"
+  }
+}
+```
+
+With the current directory in the same repository and the CLI in workload set update mode, the `install`, `update`, and `restore` commands install workloads for the specified workload set version. If you don't have a global.json file, and you're in workload set update mode, the `restore` command installs the workload set version that was established when you switched from manifests update mode to workload sets update mode.
+If you're using SDK 9, you defaulted to workload set update mode, and you might not have established a workload-set version. In that case, `restore` installs the latest workload set available on the configured feeds.
+
+If you have a workload-set version in the global.json file, the workload commands are in workload-set mode even if you haven't run the `config` command or used `--version`. The global.json file overrides those.
+To use the `--version` option in that case, run the command outside of the path containing the global.json file.
+
+If you don't specify the workload set version in global.json, you can use the `--version` option with the `restore` comand. In that case, the `restore` command selects workload set update mode before it restores workloads to the specified workload set version.
+
+In manifests update mode, `restore` installs or updates workloads to the latest version of each individual workload.
+
+## Check the update mode and version
+
+To see the current update mode, run the `config` command with the `--update-mode` option without an argument. The mode is either workload-set` or `manifests`. For example:
+
+```dotnetcli
+dotnet workload config --update-mode
+```
+
+```output
+workload-set
+```
+
+To see the current workload set version, run `dotnet workload --version`.  If a workload set is installed, you see a version such as 9.0.100-preview.7.24414.1 or 8.0.402. For example:
+
+```dotnetcli
+dotnet workload --version
+```
+
+```output
+9.0.100-preview.7.24414.1
+```
+
+In manifests mode, or if the workload set version isn't established yet after switching to workload set update mode, you see a version in the form of `<feature band>-manifests.<hash>`. For example:
+
+```dotnetcli
+dotnet workload --version
+```
+
+```output
+9.0.100-manifests.cf958b56
+```
+
+## Workload-set release cadence
+
+Workload sets are published to nuget.org with each release of the .NET SDK, under the package ID `Microsoft.NET.Workloads.<feature band>`. If, for example, you're using an 8.0.4xx SDK, the best workload set to choose is the same 8.0.4xx version of workload set.
```

Comment:
@marcpopMSFT @baronfel Is this paragraph accurate? Sometimes packages having names that fit this pattern are workload sets, and sometimes they aren't:

* Microsoft.NET.Workloads.8.0.400 is a package and you can install workload set version 8.0.400
* Microsoft.NET.Workloads.9.0.100-preview.7 is a package, but if you try to install 9.0.100-preview.7 or 9.100.0-preview.7.24414.1 (latest version) you get a not found message

In the absence of the future CLI workload set versions list command, how can people know what workload set versions are available?

Reply:
For a stable version of the SDK, we plan on always having a matching version. So 8.0.400 sdk can install 8.0.400 workload set, 401 can install a 401 set.

As you've noticed, previews are harder. One think we've done is updated the readme in the package to have the workload set info. So for the preview 7, we include that here: https://www.nuget.org/packages/Microsoft.NET.Workloads.9.0.100-preview.7/9.100.0-preview.7.24414.1#readme-body-tab

and now that you've made me look, even that is wrong and I need to go fix it as it has an extra .0. So that'll be fixed in future workload set releases but broken for 8.0.400 and 9.0.100-preview.7